src/cloud/docker/docker-mftf.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Magento Functional Testing
+group: cloud-guide
+functional_areas:
+  - Cloud
+---
+
+Magento CLoud Docker supports Functional Testing with [Magento Functional Testing Framework]({{site.baseurl}}/mftf/docs/introduction.html)
+
+This means that you're able to run Functional Tests locally and target them on your Cloud Docker or remote instance.
+
+{:.bs-callout-info}
+To be able to run MFTF use [Magento Cloud Docker][cloud-docker-repo] version 1.0 or higher
+
+1. Prepare environment
+
+    - Add MFTF dependency via Composer tou your project,
+
+    ```bash
+    composer require "magento/magento2-functional-testing-framework" --no-update
+   ```
+
+   - Install new Composer Dependency
+   
+   ```bash
+    composer update
+    ```
+
+1. Generate `docker-compose.yml` file
+
+    ```bash
+    ./vendor/bin/ece-docker build:compose --with-selenium --no-cron
+    ```
+
+1.  Start Cloud Docker
+
+    Optionally you can setup Cloud Docker to work in [Developer Mode]({{site.baseurl}}/cloud/docker/docker-mode-developer.html)
```

Comment:
```suggestion
    Optionally, you can start Cloud Docker in [Developer mode]({{site.baseurl}}/cloud/docker/docker-mode-developer.html)
```

Reply:
So the reason to use developer mode is to have a writable environment?  If we provide the option, it might be helpful if this option has a brief explanation about why a customer would want to use developer mode instead of production mode. If it's too complex to explain here, then just pointing them to the developer mode section is OK.